circle.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       # now install and checks the resulting file (just in case)
       - run:
           command: |
-            dnf install -y meson python3-devel pygobject3-devel
+            dnf install -y meson gettext python3-devel pygobject3-devel
```

Comment:
this patch needs to be rebased to before when we need it (or just squashed in with the first commit that needs it)